src/compressor.h

```diff
@@ -6,14 +6,21 @@
 #ifndef BITCOIN_COMPRESSOR_H
 #define BITCOIN_COMPRESSOR_H
 
+#include <prevector.h>
 #include <primitives/transaction.h>
 #include <script/script.h>
 #include <serialize.h>
 #include <span.h>
 
-bool CompressScript(const CScript& script, std::vector<unsigned char> &out);
+/**
+ * This saves us from making many heap allocations when serializaing
+ * and deserializaing compressed scripts
+ */
+typedef prevector<33, unsigned char> CCompressedScript;
```

Comment:
nit: How about `using CompressedScript = prevector<33, unsigned char>;`

"Do not prefix class names with `C`."
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c

Reply:
ok I was just following code I saw in the codebase, will update

Reply:
Yeah the codebase is a mix of styles because we don't update code unless we have a need to touch it - updating the codebase wholesale would introduce risk, conflicts between pull requests, and make it more difficult to properly scrutinize changes to the code over time.